src/System.Management.Automation/engine/Modules/TestModuleManifestCommand.cs

```diff
@@ -131,6 +131,22 @@ protected override void ProcessRecord()
                             }
                         }
 
+                        //RootModule can be null, empty string or point to a valid .psm1, or .dll.  Anything else is invalid.
```

Comment:
`cdxml` and `xaml` modules are allowed in `RootModule` just as they are allowed in `NestedModules`.